### DISCUSSION TITLE
Class Fan Out Complexity exclude packages based on regex

### DISCUSSION THREAD

**Discussion by sudhu678** [checkstyle/checkstyle#15833] [Q&A]
We have project built using spring boot framework and other third party libraries. Is it possible to exclude classes in package and sub-packages using regex? I tried providing regex for excludeClassesRegexps property. It works only for class name and not accounting fully qualified class name.  Please let me know if there is any workaround. Excluding every package is not scalable for us.

https://checkstyle.org/checks/metrics/classfanoutcomplexity.html

**Comment by romani** [comment]
To answer your question we need more details on what you tried.
Please try to change your post to follow format of issues reporting https://github.com/checkstyle/checkstyle/blob/master/.github/ISSUE_TEMPLATE/bug_report.md
To show what you have and what you would like to do.
